lib/taskrunner.js

```diff
@@ -33,6 +33,26 @@ module.exports = {
       params.overrides.containerOverrides[0].environment = options.env;
     }
 
+    if (options.fargate !== undefined) {
+      params.launchType = 'FARGATE';
+    }
+
+    if (options.subnets !== undefined || options.securityGroups !== undefined) {
+      params.networkConfiguration = {
+        awsvpcConfiguration: {
+          assignPublicIp: 'DISABLED'
```

Comment:
why is this assumed if awsvpc networking enabled?

Reply:
added an option instead of assuming this.